CONTRIBUTING.md

```diff
@@ -51,6 +51,8 @@ Create a new [GitHub App](https://github.com/settings/apps), setting the followi
 Your new GitHub app will need the following repository permissions:
 
 + Contents: Read & write
++ Actions: Read & write
++ Deployments: Read & write
```

Comment:
What do you need to write for actions/deployments?

Reply:
I just tested it and we actually don't need it, read-only perms are enough.